erase-install.sh

```diff
@@ -1510,6 +1509,7 @@ show_help() {
 }
 
 finish() {
+	local exit_code=$?
```

Comment:
Is the extra indentation intended?

Reply:
oops, there was one tab instead of 4 spaces.

Reply:
Ah yes, that old chestnut. Looking forward to Jams Smith's MacDevOpsYVR talk on how to set up Visual Studio Code properly :)